documentation/src/main/asciidoc/topics/ref_performance_tuning.adoc

```diff
@@ -79,6 +79,92 @@ The recommended values are:
 * send window size - 640 KB
 * receive window size - 25 MB
 
+== Thread pool settings
+
+A thread pool manages threads by assigning them concurrent tasks.
+Additionally, a thread pool can improve system performance by using any idle thread instead of creating threads.
+
+Thread pools limit the creation and deletion of threads to active worker threads, so that the amount of threads does not typically exceed the amount of tasks.
+This helps reduce the memory footprint of your {brandname} server.
+
+You can configure your `threading` subsystem, so that you can better manage thread pools and their resources. 
+
+The following table outlines common `threading` subsystem components that you can configure:
+
+// Resource: https://docs.jboss.org/infinispan/13.0/configdocs/infinispan-config-13.0.html
```

Comment:
Should we use 14.0? https://docs.jboss.org/infinispan/14.0/configdocs/infinispan-config-14.0.html

Reply:
Updated and move to a link section.